Add addon hodgepodge writeup (#948)
ember-learn/ember-blog: content/the-ember-times-issue-178.md

---

> ## Addon roundup 🙌

Author: could also change to `Addon hodgepodge roundup`, if not too weird 😅

Reviewer: I think Addon roundup sounds fine. 🙂

---

## Addon roundup 🙌

We wanted to highlight some recent happenings around the web! Check them out and give them a try! 😎

<!-- alex ignore easy -->

> * Do you want to use OAuth in your Ember app? [Sinan Keskin (@sinankeskin)](https://github.com/sinankeskin) wrote a new addon, [ember-oauther](https://github.com/sinankeskin/ember-oauther), which allows you to sign in and register a user with OAuth easily! OAuth is an open standard for **access delegation**, commonly used as a way to register or log into a web app via their Facebook, Google, Microsoft, etc. login.

Reviewer: optional:

```suggestion
* Do you want to use OAuth in your Ember app? [Sinan Keskin (@sinankeskin)](https://github.com/sinankeskin) wrote a new addon, [ember-oauther](https://github.com/sinankeskin/ember-oauther), which allows you to sign in and register a user with OAuth easily! OAuth is an open standard for **access delegation**, commonly used to help a user register or log in to a web app via their account for Facebook, Google, Microsoft, etc.
```

Author: brought this over to the main branch!, feel free to commit edits anytime!
